content/rc/security/access-control/data-access-control/role-based-access-control.md

```diff
@@ -35,6 +35,10 @@ Finally, in the **Users** tab, you [create users]({{<relref "rc/security/access-
 
 {{<image filename="images/rc/data-access-control-users.png" alt="Data access control screen." >}}{{< /image >}}
 
+{{<note>}}
+Users created for data access are different from the account access users.{{</note>}}
```

Comment:
I would add a link here to the access management docs.

```suggestion
Database access users are different from account access users. To learn more, see [Access management](INSERT RELREF HERE).{{</note>}}
```

Reply:
I will fix this directly